### PR TITLE
Jws validation ds defaults

### PR DESCRIPTION
**Pull request by guidograzioli**
* Fix issue reading PID from very long service cmdlines
* Reduce redundant defaults in molecule, downstream testing
* Make jws_home really configurable

### CODE REVIEW

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -3,11 +3,7 @@
...
-  vars:
-    jws_home: "/opt/apache-tomcat-{{ tomcat_version }}"
   collections:
     - middleware_automation.jws
-  tasks:
-    - name: "Verify that state is consistent"
-      include_role:
-        name: jws_validation
+  roles:
+    - jws_validation
```
Comment (rpelisse) [inline molecule/default/verify.yml:9]: @guidograzioli keep the include_role:, I know I did for the molecule scenario to run smoothly on Olympus.

File: playbooks/playbook.yml [hunk 2]
```diff
@@ -6,6 +6,5 @@
     - vars.yml
   collections:
     - middleware_automation.jws
-  tasks:
-    - ansible.builtin.include_role:
-        name: jws
+  roles:
+    - jws
```
Comment (rpelisse) [inline playbooks/playbook.yml:10]: @guidograzioli same remark as above

File: roles/jws/tasks/install/zipfiles.yml [hunk 3]
```diff
@@ -10,11 +10,11 @@
 - name: Add zipfile to unarchive list
   ansible.builtin.set_fact:
     zipfiles:
-      - { src: "{{ zipfile_name }}", creates: "{{ jws_home }}/bin" }
+      - { src: "{{ zipfile_name }}", creates: "{{ jws.install_dir }}/{{ zipfile_rootdir }}/bin" }
 
 - name: Add native zipfile to unarchive list
   ansible.builtin.set_fact:
-    zipfiles: "{{ zipfiles + [{'src':jws_native_zipfile, 'creates':jws_home +'/lib/libtcnative-1.so'}] }}"
+    zipfiles: "{{ zipfiles + [{'src':jws_native_zipfile, 'creates': jws.install_dir + '/' + zipfile_rootdir + '/lib/libtcnative-1.so'}] }}"
```
Comment (rpelisse) [inline roles/jws/tasks/install/zipfiles.yml:17]: for some reason, jinja2 lint prefers creates':jws (no space)

File: roles/jws/templates/systemd/service.sh.j2 [hunk 4]
```diff
@@ -31,7 +31,7 @@ case "${COMMAND}" in
       options="${options} -Djava.security.manager -Djava.security.policy==${CATALINA_BASE}/conf/catalina.policy"
     fi
     {{ jws_home }}/bin/startup.sh
-    readonly PID=$(ps -F | grep org.apache.catalina.startup.Bootstrap | sed -e '/grep/d' | sed -e 's/^[^ ]* *//' -e 's/  *.*$//')
+    readonly PID=$(ps -F -ww| grep org.apache.catalina.startup.Bootstrap | grep -v grep | awk '{print $2}')
```
Comment (rpelisse) [inline roles/jws/templates/systemd/service.sh.j2:34]: @guidograzioli sed > awk!!! :)           (joke on the side :+1: )

**Review by rpelisse — CHANGES_REQUESTED** [review]
Looks Awesome to me!

File: roles/jws/templates/systemd/service.sh.j2 (hunk 4 above)
Comment (guidograzioli) [reply roles/jws/templates/systemd/service.sh.j2:34]: lmao (the "fix" is actually the `-ww` in ps to allow the output to get to the Bootstrap main class which is at the end)

and, ahem, I am sed guy

**Review by rpelisse — APPROVED** [review]

**Pull request merged**